gitopsaddon/charts/openshift-gitops-operator/templates/openshift-gitops-operator-manager-role.clusterrole.yaml

```
@@ -458,10 +457,11 @@ rules:
   - get
   - update
   - watch
-- nonResourceURLs:
-  - /metrics
+- apiGroups:
```

YAMLlint (1.37.1)

[error] empty value in block mapping

(empty-values)